payjoin-directory/src/lib.rs

```diff
@@ -267,6 +268,67 @@ async fn health_check() -> Result<Response<BoxBody<Bytes, hyper::Error>>, Handle
     Ok(Response::new(empty()))
 }
 
+async fn handle_directory_home_path() -> Result<Response<BoxBody<Bytes, hyper::Error>>, HandlerError>
+{
+    let mut res = Response::new(empty());
+    *res.status_mut() = StatusCode::OK;
+    res.headers_mut().insert(CONTENT_TYPE, HeaderValue::from_static("text/html"));
+
+    let html = r#"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Payjoin Directory</title>
+    <style>
+        body {
+            background-color: #0f0f0f;
+            color: #eaeaea;
+            font-family:  Manrope, sans-serif;
+            padding: 2rem;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        .container {
+            background: #1a1a1a;
+            border: 1px solid #333;
+            border-radius: 8px;
+            padding: 2rem;
+            box-shadow: 0 0 10px rgba(0, 170, 255, 0.2);
+            text-align: center;
+        }
+        h1 {
+            color: black;
+            background-color: #C71585;
+            margin-bottom: 1rem;
+            padding: 0.5rem;
+            border-radius: 4px;
+        }
+        p {
+            color: #ccc;
+        }
+        a{
+            color: #F75394;
+            text-decoration: none;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <h1>Payjoin Directory</h1>
+        <p>This is a mailbox for payjoin transactions, which is used to facilitate asynchronous payments between supporting Bitcoin wallets.</p>
```

Comment:
```suggestion
        <p>The Payjoin Directory provides a rendezvous point for sender and receiver to meet. The directory stores Payjoin payloads to support asynchronous communication.</p>
```

*Facilitate payments* is pretty loaded language I'm not the most comfortable with. I'd like to borrow from the BIP with as plain language as is possible https://github.com/DanGould/bips/blob/pjv2-bak/bip-0077.mediawiki#directory-interactions

The directory *has* mailboxes, it isn't itself a mailbox according to BIP 77 as well.